examples/libsodium/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+project(libsodium-example)
+
+set(HUNTER_CONFIGURATION_TYPES Release) #Release or Debug
```

Comment:
There is one issue though,with the default config, when building both configs, the library from one build type will be overwritten by the library for the other type when installing. Unless it would be ok to change the output name of the library file.